deploy/prometheus/prometheus-alert-rules.yaml

```diff
@@ -421,7 +454,336 @@ spec:
             identifier: "{{ $labels.hostname }}"
             summary: "Increased SlowRead"
             description: |-
-              `{{ $labels.hostname }}` in namespace `{{ $labels.exported_namespace }}` increase SlowRead in `system.events` table.
+              `increase(chi_clickhouse_event_SlowRead[1m])` = {{ with printf "increase(chi_clickhouse_event_SlowRead{hostname='%s',exported_namespace='%s'}[1m])" .Labels.hostname .Labels.exported_namespace | query }}{{ . | first | value | printf "%.2f" }}{{ end }}
+              `clickhouse-server` increase SlowRead in `system.events` table.
               It mean reads from a files that were slow. This indicate system overload. Thresholds are controlled by `SELECT * FROM system.settings WHERE name LIKE 'read_backoff_%'`.
               System will reduce the number of threads which used for processing queries.
               Check you disks utilization and hardware failures.
+
+    # based on https://blog.serverdensity.com/how-to-monitor-zookeeper/
+    # and https://github.com/apache/zookeeper/blob/master/zookeeper-contrib/zookeeper-contrib-monitoring/nagios/services.cfg
+    - name: ./zookeeper.rules
```

Comment:
why does the name looks like a path to a dir?

Reply:
fixed, thanks